app/connectors/EventReportConnector.scala

```diff
@@ -58,6 +61,19 @@ class EventReportConnectorImpl @Inject()(
     }
   }
 
+  override def compileEventOneReport(pstr: String, data: JsValue)
+                                        (implicit headerCarrier: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+    val compileEvent1ReportUrl = config.compileEvent1ReportUrl.format(pstr)
```

Comment:
any reason why the log is set to warning?

Reply:
Log set to warning matches previous implementation used in AFT

Reply:
better if we set it to Info or debug, not sure why is set to warn in AFT!